File: app/src/main/java/com/android/vferreirati/starwarscharacters/models/Specie.java
package com.android.vferreirati.starwarscharacters.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Specie {

    @SerializedName("name")
    @Expose
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/br/com/starwarswiki/models/Specie.kt
package br.com.starwarswiki.models

import com.google.gson.annotations.SerializedName
import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
data class Specie(
    @Id(assignable = true) var _id: Long = 1,
    @SerializedName("name") val name: String = "",
    @SerializedName("url") val url: String = ""
)